doc/changelog/03-notations/13840-print-prim.rst

```diff
@@ -0,0 +1,10 @@
+- **Changed:**weather
+  Option :cmd:`Set Printing Notations` does not control
+  if strings and numbers are printed raw anymore.
+  (`#13840 <https://github.com/coq/coq/pull/13840>`_,
+  by Enrico Tassi).-
```

Comment:
```suggestion
```suggestion
  Option :cmd:`Set Printing Notations` no longer controls
  whether strings and numbers are printed raw
  (`#13840 <https://github.com/coq/coq/pull/13840>`_,
  by Enrico Tassi).
```